PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py

```diff
@@ -12,105 +12,116 @@ def applySubstructure( process ) :
     from PhysicsTools.PatAlgos.producersLayer1.jetProducer_cfi import _patJets as patJetsDefault
 
     #add AK8
-    addJetCollection(process, labelName = 'AK8',
-                     jetSource = cms.InputTag('ak8PFJetsCHS'),
+    addJetCollection(process, postfix=postfix, labelName = 'AK8',
+                     jetSource = cms.InputTag('ak8PFJetsCHS'+postfix),
                      algo= 'AK', rParam = 0.8,
                      jetCorrections = ('AK8PFchs', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute']), 'None'),
                      genJetCollection = cms.InputTag('slimmedGenJetsAK8')
                      )
-    process.patJetsAK8.userData.userFloats.src = [] # start with empty list of user floats
-    process.selectedPatJetsAK8.cut = cms.string("pt > 170")
+    getattr(process,"patJetsAK8"+postfix).userData.userFloats.src = [] # start with empty list of user floats
+    getattr(process,"selectedPatJetsAK8").cut = cms.string("pt > 170")
 
 
     ## AK8 groomed masses
     from RecoJets.Configuration.RecoPFJets_cff import ak8PFJetsCHSPruned, ak8PFJetsCHSSoftDrop
-    addToProcessAndTask('ak8PFJetsCHSPruned', ak8PFJetsCHSPruned.clone(), process, task)
-    addToProcessAndTask('ak8PFJetsCHSSoftDrop', ak8PFJetsCHSSoftDrop.clone(), process, task)
-    process.load("RecoJets.JetProducers.ak8PFJetsCHS_groomingValueMaps_cfi")
-    task.add(process.ak8PFJetsCHSPrunedMass)
-    task.add(process.ak8PFJetsCHSTrimmedMass)
-    task.add(process.ak8PFJetsCHSFilteredMass)
-    task.add(process.ak8PFJetsCHSSoftDropMass)
-    process.patJetsAK8.userData.userFloats.src += ['ak8PFJetsCHSPrunedMass','ak8PFJetsCHSSoftDropMass']  
-    process.patJetsAK8.addTagInfos = cms.bool(False)
+    addToProcessAndTask('ak8PFJetsCHSPruned'+postfix, ak8PFJetsCHSPruned.clone(), process, task)
+    addToProcessAndTask('ak8PFJetsCHSSoftDrop'+postfix, ak8PFJetsCHSSoftDrop.clone(), process, task)
+    #process.load("RecoJets.JetProducers.ak8PFJetsCHS_groomingValueMaps_cfi")
+    #task.add(process.ak8PFJetsCHSPrunedMass)
+    #task.add(process.ak8PFJetsCHSTrimmedMass)
+    #task.add(process.ak8PFJetsCHSFilteredMass)
+    #task.add(process.ak8PFJetsCHSSoftDropMass)
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant